docs/Home.md

```diff
@@ -1,58 +1,72 @@
-Welcome to the Terasology Wiki!
+<div class="img-responsive">
+  <img src="/images/terasology-logo.png" alt="Terasology Banner" width="300">
+</div>
 
-This is a wiki for those interested in contributing to the development of the project.
 
-It doesn't cover game content or how to play the game much, but check out the docs on [Playing](https://github.com/MovingBlocks/Terasology/blob/develop/docs/Playing.md) (including hotkeys etc) and [Modules](https://github.com/MovingBlocks/Terasology/blob/develop/docs/Modules.md) for some of that.
+# Welcome to Terasology!
 
-Use the sidebar to arrive at the most central topics this wiki covers.
+🌟 **Embark on an open-source adventure** where you can contribute, play, and explore the world of Terasology.
 
-See [What is Terasology](What-is-Terasology.md) for some more background information.
 
-## Joining the Community & Asking for Help
+Whether you're here to join the community, contribute to the project, or just learn more about the game, we've got you covered.
 
-Our main place of communication is on our [Discord Server](https://discord.gg/terasology).
-Make sure to check in, introduce yourself and what you're interested in with regards to Terasology :wave:
-For any playing related issues, leave us a note in the `#play-terasology` channel.
-Troubleshooting workspace setup, compile / test issues or other development related issues can be raised in the `#terasology` channel or by [opening an issue on this repo](https://github.com/MovingBlocks/Terasology/issues/new/choose).
+## Understanding Terasology
 
-Our [forum](https://forum.terasology.org/forum/) is currently mainly used to track progress of our GSoC student projects.
-However, it has a lot of more or less actionable ideas for improvement and a bunch of history of our current gameplays floating around, so feel free to roam around a bit and get inspired :wink:
+Terasology is a mosaic of many components, beautifully combined to create a sandbox for your imagination.
 
-## Contributing
+- 🏛️ **[Architecture Overview](Codebase-Structure.md)**
+  - Understand the high-level structure of the project.
+- 🔧 **[Entity System Architecture](Entity-System-Architecture.md)**
+  - Dive into how our entity system powers the game.
+- 🎲 **[Events and Systems](Events-and-Systems.md)**
+  - Explore how to inject new logic and life into the game.
+- 🏗️ **[Block Architecture](https://github.com/Terasology/TutorialAssetSystem/wiki/Block-Attributes)**
+  - Delve into the building blocks of the game world.
 
-Interested in getting involved with working on the game? Make sure to check out the [Contributor Quick Start](Contributor-Quick-Start.md) for setting up your first workspace and starting the game from source. It also has useful links on how to start with your first contribution.
+![Terasology - High-Level Architecture](architecture.png)
```

Comment:
This image alone looks a bit out of place with the new page content. It might be better off being moved to the codebase structure page now.

Reply:
should it be removed or not?